elastic_handler.py

```diff
@@ -0,0 +1,24 @@
+from elasticsearch import Elasticsearch
+import docker
+import time
+
+ELASTIC_PORT = f'9200'
+
+
+def start_elastic():
+    client = docker.from_env()
+    container = client.containers.run('elasticsearch', ports={ELASTIC_PORT: ELASTIC_PORT}, detach=True, remove=True)
+    # Wait for elastic container to start.
+    logs = container.logs(stream=True, tail=2)
```

Comment:
Nice! But problematic, because these logs can be changed (even in a minor version).
There's a better way to do it, specifically with Elasticsearch, by using the `ping()` method in their Python client.

Reply:
Cool. Fixed